### PR TITLE
Make the error messages more expressive

### PR DESCRIPTION
**Pull request by kytrinyx**
@stkent Would you take a look at the new error messages? Examples can be found in the evaluate_test.go file in this PR.

### CODE REVIEW

**Comment by stkent** [comment]
@kytrinyx brilliant! I'll update #20 in a sec to mirror this error language more closely.

File: evaluate.go [hunk 1]
```diff
@@ -27,23 +27,23 @@ func Evaluate(path string) bool {
 	configErrors := []ConfigError{
 		{
 			check: track.MissingProblems,
-			msg:   "-> No directory found for %v.\n",
+			msg:   "-> An exercise named '%v' is referenced in config.json, but no implementation was found.\n",
```
Comment (stkent) [inline evaluate.go]: Maybe `An exercise named` -> `An exercise with slug`?

Comment (stkent) [reply evaluate.go]: Or just `'%v' is referenced in config.json`...?

Comment (kytrinyx) [reply evaluate.go]: Oh, good point! An exercise with slug is much better. Let me change that.

Comment (kytrinyx) [reply evaluate.go]: Fixed!

**Pull request merged**